meerk40t/gui/wxmmain.py

```diff
@@ -1757,6 +1768,25 @@ def set_file_as_recently_used(self, pathname):
                 break
         self.populate_recent_menu()
 
+    def clear(self):
```

Comment:
Can we use a slightly more detailed method name than `clear` here? In my mind at least `clear` has a more generic usage than this.

Reply:
In wxmain it clears the working file, all the elements and the laserpath in the background. I'm not sure clear is wrong here. There shouldn't be any work left.